types/trigger.d.ts

```diff
@@ -95,7 +100,7 @@ export type BaseTrigger<Data> = {
   alarmText?: TriggerField<Data, TriggerOutput<Data, MatchesAny>>;
   alertText?: TriggerField<Data, TriggerOutput<Data, MatchesAny>>;
   infoText?: TriggerField<Data, TriggerOutput<Data, MatchesAny>>;
-  tts?: TriggerField<Data, TriggerOutput<Data, MatchesAny>>;
+  tts?: TriggerField<Data, PartialTriggerOutput<Data, MatchesAny>>;
```

Comment:
Should this be the type in `ResponseOutput` above as well? It seems like it'd be surprising if somebody updates responses to have custom tts for some language for some response and it behaved differently.